packages/patterns-react/src/patterns/ListSection/List.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import PropTypes from 'prop-types';
+import { ArrowRight20 } from '@carbon/icons-react';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * List Component
+ *
+ * @param {object}  props props object { lists:{ title, copy, link: { href, target}} }
+ * @param {string} props.lists.title List component title property
+ * @param {string} props.lists.copy List component copy property
+ * @param {object} props.lists.link List Component link object
+ * @param {string} props.lists.link.href List Component href property of link object
+ * @param {string} props.lists.link.target List Component target property of link object
+ * @returns {object} JSX Object
+ */
+const List = ({
+  lists: {
+    title,
+    copy,
+    link: { href, target },
+  },
+}) => {
+  return (
+    <div
+      data-autoid={`${stablePrefix}--list`}
```

Comment:
Thinking we should possibly call this as something that's underneath the listsection, perhaps `dds-listsection-item`?
cc: @photodow @annawen1 @andysherman2121

Reply:
yeah that makes sense to me

Reply:
yeah agree. I will change it.

Reply:
Renamed List component to ListSectionItem component by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/573/commits/da28718a9bb0eb11f93d2d76d39ec1d34fc1576e)